source/common/network/listener_impl.h

```diff
@@ -17,13 +17,14 @@ namespace Network {
 class ListenerImpl : public Listener {
 public:
   ListenerImpl(Event::DispatcherImpl& dispatcher, ListenSocket& socket, ListenerCallbacks& cb,
-               bool bind_to_port);
+               bool bind_to_port, bool hand_off_restored_destinations);
 
 protected:
   virtual Address::InstanceConstSharedPtr getLocalAddress(int fd);
 
   Address::InstanceConstSharedPtr local_address_;
   ListenerCallbacks& cb_;
+  bool hand_off_restored_destinations_;
```

Comment:
const

Reply:
I have to learn this someday :-) Btw. this factors in to the discussion about this being plural or not in the other comment..